sdl2_image/VITABUILD

```diff
@@ -16,7 +16,7 @@ prepare() {
 build() {
   cd SDL2_image-${pkgver}
   mkdir build && cd build
-  cmake .. -DCMAKE_TOOLCHAIN_FILE=$VITASDK/share/vita.toolchain.cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_POSITION_INDEPENDENT_CODE=OFF -DBUILD_SHARED_LIBS=OFF -DSDL2IMAGE_SAMPLES=OFF
+  cmake .. -DCMAKE_TOOLCHAIN_FILE=$VITASDK/share/vita.toolchain.cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_POSITION_INDEPENDENT_CODE=OFF -DBUILD_SHARED_LIBS=OFF -DSDL2IMAGE_SAMPLES=OFF -DSDL2IMAGE_WEBP=ON -DSDL2IMAGE_DEPS_SHARED=OFF
```

Comment:
Does this change require the depends to be updated to include webp?

Reply:
It's already depends on libwebp